src/Symfony/Bundle/WebProfilerBundle/Tests/EventListener/TestStreamedResponseHtml.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symfony\Bundle\WebProfilerBundle\Tests\EventListener;
+
+use Symfony\Component\HttpFoundation\HtmlResponseInterface;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+/**
+ * @internal
+ *
+ * @author Bartłomiej Krukowski <bartlomiej@krukowski.me>
+ */
+class TestStreamedResponseHtml extends StreamedResponse implements HtmlResponseInterface
+{
+    private $extraHtml = '';
+
+    public function appendToBody($html)
+    {
+        $this->extraHtml .= $html;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function sendContent()
+    {
+        ob_start();
+        parent::sendContent();
+        $content = ob_get_contents();
+        ob_end_clean();
```

Comment:
what @stof means is that the test should reflect what you are trying to implement, so it must be a streamed response.